extensions/cornerstone-dicom-sr/src/commandsModule.js

```diff
@@ -29,15 +29,13 @@ const _generateReport = (
 
   const report = MeasurementReport.generateReport(
     filteredToolState,
-    metaData,
-    utilities.worldToImageCoords
+    OHIF.classes.MetadataProvider,
```

Comment:
metaData was correct since it uses .get and it will act based on priority. Why you are forcing it to use the OHIF one? it should be based on priorities like elseWhere